Dockerfile

```diff
@@ -1,16 +1,22 @@
-FROM docker.io/alpine:edge AS build
-WORKDIR /dms
-ADD . /dms
-RUN apk add --no-cache go gcc musl-dev
-RUN go mod tidy 
-RUN go build -trimpath -buildmode=pie -ldflags="-s -w" -o dms
+FROM docker.io/golang:1.19-alpine AS build
 
+WORKDIR /build
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
```

Comment:
Why CGO_ENABLED=0? One advantage of building inside the image is we can enable this.